sounddevice_client.py

```diff
@@ -0,0 +1,52 @@
+"""
+Simple (hopefully) multiplatform client mimicing the linux command
+
+    arecord -f S16_LE -c1 -r 16000 -t raw -D default | nc localhost 43001
+
+which streams audio data from the microphone to the server.
+Tested on Mac Os.
+"""
+import sys
+import sounddevice as sd
+import socket
+import numpy as np
+from argparse import ArgumentParser
+
+
+parser = ArgumentParser(__doc__)
+parser.add_argument('--host', type=str, default='localhost', help='Host name')
+parser.add_argument('--port', type=int, default=43007, help='Port number')
+parser.add_argument('--chunk', type=int, default=1000, help='Chunk size in ms')
```

Comment:
just a hint: there is also another variable called chunk with another meaning. maybe we can rename this one to a more speaking one which indicates its about MS?